website/docs/reference/widgets/json-form.md

```diff
@@ -32,6 +32,8 @@ These properties allow you to edit the JSON Form widget. All of these properties
 | **Animate Loading**       | When turned off, the widget will load without any skeletal animation. You can use a toggle switch to turn it on/off. You can also turn it off/on using javascript by enabling the JS label next to it.                       |
 | **Fixed Footer**          | Makes the footer sticky for long forms, so the Submit and Reset buttons are always visible on long forms.                                                                                                                    |
 | **Visible**               | Controls widget's visibility on the page. When turned off: The widget will not be visible when the app is published. It appears translucent when in Edit mode.                                                               |
+| **Hidden Fields in Data**               | When turned on, the output data will also contain data from hidden fields. The hidden field values will be reference the source data.                                                               |
```

Comment:
some messup in markup . rest looks good and we merge once this is fixed

<img width="975" alt="Screenshot 2023-01-25 at 18 58 29" src="https://user-images.githubusercontent.com/32227135/214575742-42148979-2ff7-4fc3-ad20-eb9cc551af9f.png">